### ISSUE TITLE
Add missing a11y properties

### ISSUE THREAD

**Issue by untemps** [untemps/svelte-use-tooltip#11]
Implement accessibility best practices for the tooltip: [https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)
